.github/dependabot.yaml

```diff
@@ -0,0 +1,58 @@
+version: 2
+updates:
+  #upgrades production dependencies of GitHub Actions
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
+      time: "08:00"
+      timezone: America/New_York
+    assignees:
+      - Kurt-von-Laven
+    reviewers:
+      - Kurt-von-Laven
+    open-pull-requests-limit: 1
+    allow:
+      - dependency-name: actions/checkout
+      - dependency-name: actions/setup-python
+      - dependency-name: slackapi/slack-github-action
+    commit-message:
+      prefix: "fix:"
+      include: scope
+
+    #upgrades development dependencies of GitHub Actions
+    - package-ecosystem: github-actions
```

Comment:
YAML, like Python, is whitespace-sensitive, so I believe the indentation error on this line is causing issues.